Rakefile

```diff
@@ -8,5 +8,11 @@ require_relative "config/application"
 Rails.application.load_tasks
 
 if %w[development test].include?(Rails.env)
-  task default: ["lint:ruby", "lint:javascript", "lint:scss", :spec]
+
+  desc "Runs JS unit tests with yarn"
+  task javascript_specs: :environment do
+    system "yarn test"
```

Comment:
Do we need to add something to the README re installing jest globally on your local machine?

Reply:
It should be installed when you run yarn as we've declared it in the project as a dev dependency.